pkg/controllers/validationController_test.go

```diff
@@ -3,19 +3,27 @@ package controllers
 import (
 	_ "embed"
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"strings"
 	"testing"
 
+	"github.com/datreeio/admission-webhook-datree/pkg/clients"
 	"github.com/datreeio/admission-webhook-datree/pkg/config"
+	"github.com/datreeio/admission-webhook-datree/pkg/services"
+	"github.com/datreeio/datree/pkg/httpClient"
+	"github.com/datreeio/datree/pkg/networkValidator"
 	"github.com/stretchr/testify/assert"
 	admission "k8s.io/api/admission/v1"
 )
 
 //go:embed test_fixtures/applyNotAllowedRequest.json
 var applyRequestNotAllowedJson string
 
+//go:embed test_fixtures/mockServerResponse.json
+var applyRequestNotAllowedMockedResponseBody []byte
```

Comment:
I think a better name for this variable and for the file would be: `getPrerunDataResponse`